### ISSUE TITLE
Is Pandas slicing FutureWarning a bug that needs to be fixed?

### ISSUE THREAD

**Issue by rowantoday** [lspatial/baggingrnet#1]
Professor Li,

I tried to use baggingrnet for Spatiotemporal Estimation of PM2.5, a FatureWarning occurred as follows. Can this problem be fixed?

Starting process 0, pid=34300
Epoch 00096: early stopping
F:\liuzhang\code2\venv2\lib\site-packages\pandas\core\series.py:1143: FutureWarning: 
Passing list-likes to .loc or [] with any missing label will raise
KeyError in the future, you can use .reindex() as an alternative.

**Comment by rowantoday** [comment]
My software version is configured as follows.
pandas 0.25.1
tensorflow 1.12.3
keras 2.2.5

**Comment by lspatial** [comment]
Hi, may you use the early version of pandas? 
Or try to install the python package: https://pypi.org/project/baggingrnet. 
If still problems exist, please let me know. Thanks.
